packages/block-editor/src/components/block-list/block.js

```diff
@@ -183,31 +183,31 @@ function BlockListBlock( {
 					<>
 						{ blockEdit }
 						{ mode === 'html' && (
-							<Block.div __unstableIsHtml>
+							<block.Div __unstableIsHtml>
```

Comment:
As noted in the PR description, the `Div` is actually the component, not `block`.

Reply:
And `block.Div` is shorthand for `block( 'div' )` (which must be added on a separate line outside the component).